### PR TITLE
Update Windows install guide

### LINKED ISSUE #379: Update the install guide on Windows
http://flight-manual.atom.io/getting-started/sections/installing-atom/#installing-atom-on-windows

This section states that the `Open With Atom` and `Open with...` associations are added by the installer. These associations are now controlled by settings in the settings-view:

image

We need to revise this section to be up to date with the current defaults on Windows and add documentation for the System panel in the settings-view in a separate Windows-specific block.

PR that added the System Settings panel: https://github.com/atom/settings-view/pull/811

### PR DESCRIPTION
**Pull request by schalkneethling**
@Ben3eeE - Apologies for taking so long, but here it is. 🎐 

Fixes #379

### CODE REVIEW

File: content/getting-started/sections/installing-atom.md [hunk 1]
```diff
@@ -56,9 +56,13 @@ To install the `atom` and `apm` commands, run "Window: Install Shell Commands" f
 
 #### Installing Atom on Windows
 
-Atom is available with a Windows installer that can be downloaded from https://atom.io or from [Atom Releases][releases] named `AtomSetup.exe`. This setup program will install Atom, add the `atom` and `apm` commands to your `PATH`, create shortcuts on the desktop and in the start menu, add an "Open with Atom" context menu in the Explorer and make Atom available for association with files using "Open with...".
+Atom is available with a Windows installer that can be downloaded from https://atom.io or from [Atom Releases][releases] named `AtomSetup.exe`. This setup program will install Atom, add the `atom` and `apm` commands to your `PATH`, and create shortcuts on the desktop and in the start menu.
 
-![Atom on Windows](../../images/windows.gif)
+![Atom on Windows](../../images/windows-system-settings.png)
+
+The installer used to automatically add an "Open with Atom" context menu in Explorer and make Atom available for association with files using "Open with...". This is now controlled by the System Settings panel as seen above.
```
Comment (50Wliu) [inline content/getting-started/sections/installing-atom.md]: I don't think there's much merit in saying what the installer used to do.  Almost everyone who's looking at this page will be installing Atom and doesn't care about previous versions.

File: content/getting-started/sections/installing-atom.md [hunk 2]
```diff
@@ -56,9 +56,13 @@ To install the `atom` and `apm` commands, run "Window: Install Shell Commands" f
...
+
+With Atom open, click on `File > Settings`, and the the `System` tab on the left. Check the boxes next to `Show in file context menus`, as well as `Show in folder context menus`. And you’re all set.
```
Comment (Ben3eeE) [inline content/getting-started/sections/installing-atom.md]: Typo: `the the`. Did you intend to write `then the´?

**Comment by schalkneethling** [comment]
Thanks @50Wliu and @Ben3eeE for the review ~ updated.

**Comment by schalkneethling** [comment]
@Ben3eeE ~ I wonder if the pull request needs to `hacktoberfest` label?

**Comment by Ben3eeE** [comment]
> I wonder if the pull request needs to hacktoberfest label?

PRs without the hacktoberfest label will also count towards the hacktoberfest PRs. Only PRs that are closed and given an invalid label will not count.

**Comment by Ben3eeE** [comment]
Thanks for contributing :bowing_man:

**Pull request merged**